lib/roadie/style_block.rb

```diff
@@ -5,28 +5,48 @@ module Roadie
   # A style block is the combination of a {Selector} and a list of {StyleProperty}.
   class StyleBlock
     extend Forwardable
-    attr_reader :selector, :properties
+    attr_reader :selector, :properties, :media
 
     # @param [Selector] selector
     # @param [Array<StyleProperty>] properties
-    def initialize(selector, properties)
+    # @param [Array<Symbol>] media  Array of media types, e.g.
+    #                          @media screen, print and (max-width 800px) will become
+    #                          [:screen, :"print and (max-width 800px)"]
```

Comment:
It's weird to keep symbols that look like that. I'd prefer of we converted the symbols we get from css_parser into Strings before storing them in Roadie's internal models.